packages/bodiless-core/src/components/PageEditor.tsx

```diff
@@ -87,13 +88,17 @@ const PageEditor: FC<Props> = ({ children, ui }) => {
   };
 
   const { PageOverlay = () => null } = newUI;
+  console.log('pageditor');
 
   // Register buttons to the main menu.
   useRegisterMenuOptions({
     getMenuOptions,
     name: 'Editor',
   });
-  useEffect(() => { if (!context.isActive) context.activate(); }, []);
+  useEffect(() => {
+    console.log('effect', context, context.isActive);
```

Comment:
This one also.

Reply:
@lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!